lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/sys.rb

```diff
@@ -1093,7 +1099,7 @@ def cmd_reg_help
     print_line("    createkey   Create the supplied registry key  [-k <key>]")
     print_line("    deletekey   Delete the supplied registry key  [-k <key>]")
     print_line("    queryclass  Queries the class of the supplied key [-k <key>]")
-    print_line("    setval      Set a registry value [-k <key> -v <val> -d <data>]")
+    print_line("    setval      Set a registry value [-k <key> -v <val> -d <data>]. Use 2-digit hex string to set binary data with REG_BINARY type (e.g. fe340ead)")
```

Comment:
When you say 2 digit hex string most people think of "\x2D" or similar not this binary blob. I would suggest updating this eventually to support 2 digit hex strings as well via a conversion process or similar however for the point of getting this PR in, I would suggest rewording this as shown below to avoid confusion.

```suggestion
    print_line("    setval      Set a registry value [-k <key> -v <val> -d <data>]. Use binary number to set binary data with REG_BINARY type (e.g:  -v fe340ead)")
```